<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlotMonitor</class>
 <widget class="QMainWindow" name="SlotMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LocoNet Slot Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="chkShowUnusedSlots">
        <property name="toolTip">
         <string>if checked, even empty/idle slots will appear</string>
        </property>
        <property name="text">
         <string>Show Unused slots</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkShowSystemSlots">
        <property name="toolTip">
         <string>if checked, slots reserved for system use will be shown</string>
        </property>
        <property name="text">
         <string>Show System Slots</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="estopAllButton">
        <property name="text">
         <string>estop All</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="JTable" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JTable</class>
   <extends>QTableView</extends>
   <header location="global">jtable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
